src/main/java/com/puppycrawl/tools/checkstyle/PackageObjectFactory.java

```diff
@@ -239,463 +245,66 @@ private Object createObject(String className) throws CheckstyleException {
     }
 
     /**
-     * Fill short-to-full module names map.
-     */
-    private static void fillShortToFullModuleNamesMap() {
```

Comment:
hardcoded list should stay to avoid bruteforce search, is is a problem to keep it ?